modules/core/src/main/scala/lucuma/odb/api/repo/ObservationRepo.scala

```diff
@@ -297,6 +299,23 @@ object ObservationRepo {
 
       }
 
+      override def bulkEditScienceRequirements(edit: ScienceRequirementsModel.BulkEdit): F[List[ObservationModel]] = {
+        val update =
+          for {
+            vos <- TableState.observation.lookupAllValidated[State[Tables, *]](edit.observationIds)
+            os  <- (vos, edit.scienceRequirements.editor).mapN { (os, ed) =>
+                     val os2 = os.map {
+                       ObservationModel.scienceRequirements.modify { cs => ed.runS(cs).value }
+                     }
+                     Tables.observations.mod(_ ++ os2.map(o => o.id -> o)).as(os2)
```

Comment:
For what it's worth, which is probably just about nothing, `os2.fproductLeft(_.id)` is another option

Reply:
Right, will fix that